### PR TITLE
Use Go's embed feature instead of statik

### PR DESCRIPTION
**Pull request by nopcoder**
Reduce the dependencies and build complexity.
Currently we use 'statik' a command line tool to build go code to embed our UI and migration ddl files.
Using the Go's embed feature we can skip the code generation step (UI build is still required) to build.
Updated the way or UI handle identify page not found and redirect temporary.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -65,7 +65,6 @@ clean:
 		$(UI_BUILD_DIR) \
 		pkg/actions/mock \
 		pkg/api/lakefs.gen.go \
-		pkg/ddl/statik.go \
 		pkg/graveler/sstable/mock \
 		pkg/webui \
```
Comment (arielshaqed) [inline Makefile:69]: I *think* this should also be removed... but I am not sure.  Could you either explain what goes in each `webui` directory, or remove this line and `fgrep -R webui` to be sure we got all bits?

**Review by arielshaqed — APPROVED** [review]
Thanks!

Please let's have someone from core/OSS (name TBD) team review this -- it's a fairly major change to the build process.

File: Makefile (hunk 1 above)
Comment (nopcoder) [reply Makefile:69]: Right, I missed this one - thanks!

Comment (nopcoder) [reply Makefile:69]: Removing the extra directory - it was left for the cleanup and it is no longer used.
We will keep content.go file under the real webui to include the UI dist folder.

**Comment by nopcoder** [comment]
Adding @itai-david to review this change

**Comment by nopcoder** [comment]
Thanks @idanovo for helping with debugging the ui handler change with lakectl doctor.

**Review by idanovo — APPROVED** [review]
Looks good! one small question/fix

File: cmd/lakectl/cmd/dbt.go [hunk 2]
```diff
@@ -186,7 +188,7 @@ generate_schema_name.sql
 {%- endmacro %}
 `
 		//nolint:gosec
-		err := ioutil.WriteFile(macroPath, []byte(generateSchemaData), 0644)
+		err := ioutil.WriteFile(macroPath, []byte(generateSchemaData), 0o644)
```
[the remark this answers is not in the document]
Comment (nopcoder) [reply cmd/lakectl/cmd/dbt.go:191]: its an alternative way to write a number which is a file permission in Go - like: 10000 can be 10_000. In this case I think its my file formatting tool tries to make me look cool.

[the remark this answers is not in the document]
Comment (nopcoder) [reply cmd/lakectl/cmd/dbt.go:191]: Sure, will be good as a separate PR

File: pkg/db/migration.go [hunk 3]
```diff
@@ -52,12 +52,8 @@ func (d *DatabaseMigrator) Migrate(ctx context.Context) error {
 }
 
 func getStatikSrc() (source.Driver, error) {
```
Comment (itai-david) [inline pkg/db/migration.go:54]: Maybe we can rename it, now that statik is out?

**Review by itai-david — APPROVED** [review]
One minor comment regarding possible func renaming. Other than that LGTM

File: pkg/db/migration.go (hunk 3 above)
Comment (nopcoder) [reply pkg/db/migration.go]: good idea - updated, thanks.

**Pull request merged**